src/jquery.multi-select.js

```diff
@@ -100,55 +130,91 @@
         if (key === escapeKey) {
           _this.menuHide();
         }
-      });
+      })
+      .appendTo(this.$container);
 
-      this.$menu.on('change.multiselect', function() {
-        _this.updateButtonContents();
-      });
+      this.constructMenuItems();
+
+      if ( this.settings.presets ) {
+        this.constructPresets();
+      }
+    },
+
+    constructMenuItems: function() {
```

Comment:
Nah, it's fine. Would be good if closure compiler could spot this and inline, but doesn't look like it does, oh well.